### DISCUSSION TITLE
Feature GitHub discussions at ziku.la

### DISCUSSION THREAD

**Discussion by Guite** [zikula/core#4569] [Ideas]
How about integrating GH discussions at https://ziku.la/en/support and dropping Stack Overflow afterwards?

**Comment by craigh** [comment]
seems ok to me, but I like slack for conversations. maybe we leave slack active but don't advertise it? (take it off the website)

**Reply by Guite** [reply]
Of course we should keep Slack.
But I think it would be beneficial to drop SO.

**Reply by craigh** [reply]
I read it wrong 🧐

sure - drop Stack Overflow.

**Comment by Guite** [comment]
To show latest conversations on the website one of these would be useful:

1) Discussions RSS feed: https://github.com/github/feedback/discussions/31

2) Discussions API: https://github.com/github/feedback/discussions/43

**Comment by Guite** [comment]
This is done, we are using GraphQL for it.
